server/index.js

```diff
@@ -29,6 +29,34 @@ app.get('/api/products', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products/:productId', (req, res, next) => {
+  const { productId } = req.params;
+  const sql = `
+    select *
+    from "products"
+    where "productId" = $1
+  `;
+
+  const params = [productId];
+  if (productId > 6 || productId <= 0) {
```

Comment:
You shouldn't hard-code the `6` here. More products may be added later, and you don't want to have to edit the code every time somebody changes the catalog.